python/tank/deploy/tank_commands/clone_configuration.py

```diff
@@ -90,6 +90,8 @@ def run_noninteractive(self, log, parameters):
                          computed_params["path_mac"], 
                          computed_params["path_win"])
 
+        self.tk.log_metric("clone config")
```

Comment:
Maybe we could add a generic metrics logger here?: https://github.com/shotgunsoftware/tk-core/blob/master/python/tank/deploy/tank_command.py#L307